book/src/keymap.md

```diff
@@ -291,6 +291,7 @@ undo-able "save point" until you return to normal mode.
 | -----                     | -----------                 | -------                 |
 | `Escape`                  | Switch to normal mode       | `normal_mode`           |
 | `Ctrl-x`                  | Autocomplete                | `completion`            |
+| `Ctrl-s`                  | Signature help (**LSP**)    | `signature_help`        |
```

Comment:
Beside what @David-Else mentioned, `ctrl-s` might be better to leave it for https://github.com/helix-editor/helix/pull/2115#issuecomment-1107958417 to keep it similar to what it does on normal mode